planner/core/rule_build_key_info.go

```diff
@@ -91,6 +94,24 @@ func (p *LogicalSelection) checkMaxOneRowCond(unique expression.Expression, cons
 	return okCorCol
 }
 
+// isUnique Because equivalence conditions can filter out null values,so a unique index without a NotNullFlag can return at most one row.
```

Comment:
We should make the comment like a normal sentence.

`isUnique` checks the unique index whose `NotNullFlag` is false since ....

Reply:
OK，done